phoenix-core/src/it/java/org/apache/phoenix/end2end/RowValueConstructorOffsetIT.java

```diff
@@ -940,4 +941,70 @@ public void testOffsetExplain() throws SQLException {
             assertTrue(explainStringBuilder.toString().contains("With RVC Offset"));
         }
     }
+
+    @Test public void testGlobalIndexViewAccess() throws Exception {
+        String
+                ddl =
+                "CREATE TABLE IF NOT EXISTS ARCHIVE.FIELD_HISTORY_ARCHIVE (\n"
+                        + "    ORGANIZATION_ID CHAR(15) NOT NULL,\n"
+                        + "    PARENT_KEY_PREFIX CHAR(3) NOT NULL,\n"
+                        + "    PARENT_ID CHAR(15) NOT NULL,\n" + "    CREATED_DATE DATE NOT NULL,\n"
+                        + "    DATA VARCHAR   \n" + "    CONSTRAINT PK PRIMARY KEY \n" + "    (\n"
+                        + "        ORGANIZATION_ID, \n" + "        PARENT_KEY_PREFIX,\n"
+                        + "        PARENT_ID,\n" + "        CREATED_DATE\n" + "    )\n"
+                        + ") MULTI_TENANT=true";
+
+        //Index reorders the pk only
+        String
```

Comment:
nit: weird line break

Reply:
i think i used the phoenix code formatter and it did this i can see if i can trick it into formatting nicer.